addons/project/static/src/js/project_task_kanban_examples.js

```diff
@@ -15,7 +15,8 @@ const exampleData = {
     applyExamplesText: _lt("Use This For My Project"),
     examples:[{
         name: _lt('Software Development'),
-        columns: [_lt('Backlog'), _lt('Specifications'), _lt('Development'), _lt('Tests'), _lt('Delivered')],
+        columns: [_lt('Backlog'), _lt('Specifications'), _lt('Development'), _lt('Tests')],
```

Comment:
how you can keep the order if the folded was in the middle ?

Reply:
You cannot with my solution.
Do you have a use case ?

Reply:
it's a limitation, what if one day we want to do it, we can't extend your solution, we need to make many changes to achieve that.

Reply:
IMHO, we should not anticipate too much or overengineer stuff.
As of today, regarding the use cases we have, I find what I propose simpler.

Your solution is fine but I don't like its API, specifically this:
```js
[false, false, false, false, true]
```

This is not really elegant and less flexible.

Reply:
okay, i will add my team's reviewers and let them decide @xavierbol @auon-odoo

Reply:
@RaoufGhrissi 
If I am not wrong, the examples will always have the folded stages at the end. Thus, I agree with @brboi, it is not really necessary to manage all possible cases if in reality, we will only use the common use cases in the examples. 🙂 

cc @auon-odoo

Reply:
hey @xavierbol my mention was not about the order, it was about replacing my pr by this one. if you're okay i will close the other one https://github.com/odoo/odoo/pull/107633

Reply:
Hi @brboi, many thanks for your complete suggestion. Indeed, I don't see any reasonable usecase where we would have a folded stage in the middle other stages :ok_hand:

Reply:
it's useless to copy +57 −24  changes, closing mine is easier.

Reply:
@RaoufGhrissi 
I don't think I have to answer to that question 🙂 
If you agree with the PR and agree with the arguments of @brboi, then you can find yourself the answer, no? 🙂 

To be sure, we agree about that and there is no misunderstood, we (Bruno, Audric and me) did not say your solution is worst. Bruno just suggests you another solution simpler than yours since it is not needed to manage all the possible cases for the examples as we can see in your PR the stages folded in all examples are at the end. Thus, we can assume the folded stages will always be at the end, if it was not the case, then Bruno would certainly agree with your solution but it is not the case and I don't think it would be the case in the future. 🙂 

Also, `web` is their module and so the final decision belongs to them and not to Audric or me. 🙂 

If something is not clear or you do not agree about something, then we can obviously discuss in private. 🙂

cc @auon-odoo

Reply:
@RaoufGhrissi my plan was to use this PR for web only, linking yours for the changes in project.

Rationale: the changes in `web` may bring more technical debt as they are a central piece of odoo (web_read_group). They need to be carefully discussed then :-)

Reply:
Thank you very much for your help @brboi 😁

Reply:
thanks @brboi i will cherry-pick.
Clarification: i totally understand the purpose of this PR, i said okay after convincing me about the order and i just mentioned my team guru and reviewer as i though they are the persons who can say YES. i was mistaken and Xavier clarified it for me :) next time my reponse will be 'okay' simply when i'm convinced, thank you all for making the process clearer.